nengo/tests/test_learning_rules.py

```diff
@@ -70,39 +72,45 @@ def _test_pes(Simulator, nl, plt, seed,
     assert not np.allclose(weights[0], weights[-1], atol=1e-5)
 
 
-def test_pes_ens_ens(Simulator, nl_nodirect, plt, seed):
+@pytest.mark.parametrize('apply_every', [None, 1e-5, 0.005])
```

Comment:
Maybe test `apply_every=0` since it's a corner case and you might want to make it's behaviour explicit?